src/view/utils.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import ViewPosition from './position';
+import ViewTreeWalker from './treewalker';
+
+/**
+ * Contains utility functions for working on view.
+ *
+ * @module engine/view/utils
+ */
+
+/**
+ * For given {@link module:engine/view/text~Text view text node}, it finds previous or next sibling that is contained
+ * in the same container element. If there is no such sibling, `null` is returned.
+ *
+ * @param {module:engine/view/text~Text} node Reference node.
+ * @param {Boolean} getNext If `true` next touching sibling will be returned. If `false` previous touching sibling will be returned.
```

Comment:
As for this flag being optional – it's always passed in the tests: https://github.com/ckeditor/ckeditor5-engine/pull/1083/files#diff-41de3c28cf4c9630dfaec4e35850f0faR38. This should be changed also there.